src/webcomponents/variant/interpretation/variant-interpreter-grid.js

```diff
@@ -1409,11 +1413,26 @@ export default class VariantInterpreterGrid extends LitElement {
             // Generate a clone of the variant review to prevent changing original values
             this.variantReview = UtilsNew.objectClone(this.checkedVariants.get(e.currentTarget.dataset.variantId));
             this.requestUpdate();
-
-            $("#" + this._prefix + "ReviewSampleModal").modal("show");
+            const modalElm = document.querySelector("#"+this._prefix + "ReviewSampleModal");
+            UtilsNew.draggableContainer(document, modalElm);
+            $("#"+this._prefix + "ReviewSampleModal").modal({
+                backdrop: false,
+                show: true,
+            });
         }
     }
 
+    showDraggableModal(elementId) {
+        $("#" + elementId).modal({
+            backdrop: false,
+            show: true,
+        });
+
+        $(`div[id='${elementId}']>div[class*='modal-dialog']`).draggable({
+            handle: ".modal-header"
+        });
+    }
+
```

Comment:
@Rodielm why do you need this function?

Reply:
I'll remove it, it was another solution with jquery.